src/main/java/com/exadel/discount/entity/City.java

```diff
@@ -0,0 +1,32 @@
+package com.exadel.discount.entity;
+
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.hibernate.annotations.GenericGenerator;
+
+import javax.persistence.*;
+import java.util.UUID;
+
+@Data
+@NoArgsConstructor
+@Entity
+@Table(name = "cities")
+public class City {
+    @Id
+    @GeneratedValue(generator = "UUID")
+    @GenericGenerator(name = "UUID", strategy = "org.hibernate.id.UUIDGenerator")
+    private UUID id;
+
+    @Column(name = "name", nullable = false)
+    private String name;
```

Comment:
And length, please
@Column(name = "name", length = 50, nullable = false)

Reply:
Done